lms/templates/student_account/login_and_register.html

```diff
@@ -30,6 +30,7 @@
     <%static:include path="student_account/${template_name}.underscore" />
 </script>
 % endfor
+<script type="text/javascript" src="${static.url('js/onboarding_survey/registration.js')}" charset="utf-8"></script>
```

Comment:
Override `login_and_register.html` in philu's theme instead of changing in lms.

Reply:
Will be applied tomorrow.